config/crd/bases/test.openstack.org_tempests.yaml

```diff
@@ -73,6 +73,28 @@ spec:
                   corresponding Tempest CR is deleted. This allows the user to debug
                   any potential troubles with `oc rsh`.
                 type: boolean
+              extraConfigmapsMounts:
```

Comment:
Please run `make bundle` to update the test-operator CSV as well [1]. 

[1] https://github.com/openstack-k8s-operators/test-operator/blob/8a982d2f15ed27ccf88e97afaa1ad270f502f8be/config/manifests/bases/test-operator.clusterserviceversion.yaml